infra/us-east-1/prod/rds/bastion/ec2-instance/terragrunt.hcl

```diff
@@ -0,0 +1,60 @@
+locals {
+
+  # Automatically load environment-level variables
+  account_vars     = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+
+  # Extract out common variables for reuse
+  project        = local.account_vars.locals.project
+  app            = "marlowe-runtime"
+
+  bastion_name       = "${local.project}-${local.app}-database-bastion"
+
+  subnet_id              = "subnet-087617570cd189b3e"
+
+  tags = {
+    organization = "iog"
+    tribe        = "smart-contracts"
+    environment  = "prod"
+    function     = "bastion"
+    Terraform    = "true"
+    Project      = local.project
+    Resource     = local.bastion_name
+  }
```

Comment:
Please check if we can use the tags defined by the provider: 
https://github.com/input-output-hk/sc-dev-platform/blob/main/infra/terragrunt.hcl#L64